src/model/engines/pep8.nit

```diff
@@ -0,0 +1,140 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Pep/8 terminal engine for submissions
+#
+# TODO: Pep8Term is crap software to install and to execute.
+# Currently, this program assume that it is installed in the subdirectory `pep8term`.
+# Use `make pep8term` to install and compile it.
+module pep8
+
+import engine_base
+
+# The absolute path of a `file` in pep8term.
+# Aborts if not found
+fun pep8term(file: String): String do
+	var dir = "pep8term"
+	if not dir.file_exists then
+		print_error "{dir}: does not exists. Please install it (make pep8term)."
+		exit 1
+	end
+	file = dir.realpath / file
+	if not file.file_exists then
+		print_error "{file}: does not exists. Please check the pep8term installation."
+		exit 1
```

Comment:
Yup, this part puzzled me, it's still in the poc part of pep8 I left it as-is, but it should be rewritten later